app/routes/explore.js

```diff
@@ -1,29 +1,73 @@
 import Ember from 'ember';
-import moment from 'moment';
 
 const { Route, RSVP } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Explore');
   },
+  queryParams: {
+    category: {
+      refreshModel: true
+    },
+    sub_category: {
+      refreshModel: true
+    },
+    event_type: {
+      refreshModel: true
+    },
+    date_range: {
+      refreshModel: true
+    }
+  },
 
-  model() {
-    return RSVP.hash({
-      events: this.store.query('event', {
-        sort   : 'starts-at',
-        filter : [
-          {
-            name : 'starts-at',
-            op   : 'ge',
-            val  : moment().toISOString()
-          },
-          {
-            name : 'state',
+  model(params) {
+    this.set('params', params);
+    let filterOptions = [];
+    if (params.category) {
```

Comment:
`filterOptions` is an array. It can accept multiple filters. See how we're using multiple filters in events route. We're already using multiple filters in different parts of the app.